### ISSUE TITLE
[Feature Request] Implement a batch-style freeze interface 实现批量式的冻结接口

### ISSUE THREAD

**Issue by andy-math** [aistra0528/Hail#152]
由于island的open api依赖于从broadcast层面返回调用结果，因此这一类api是异步的。
目前，Hail在使用Island模式时，不得不在每个冻结请求时阻塞主线程，直到timeout或者收到执行结果。这在用户一次性冻结多个app时，将会反复阻塞主线程。
Island的open api允许一次性发送多个app包名，这样通过一次性的请求，即可在阻塞一次的情况下完成对所有app的冻结，而这无需把冻结流程完全重构为并发式的。
综上所述，请求作者实现一个批量式的冻结接口，这对island工作模式会很有帮助。

Since island's open api relies on returning the call result from the broadcast, this api is asynchronous.
Currently, Hail has to block the main thread on each freeze request until timeout or until it receives the result of the execution when using Island mode. This will repeatedly block the main thread when the user freezes multiple apps at once.
Island's open api allows multiple app package names to be sent at once, so that all apps can be frozen in a single request while blocking once, without having to completely refactor the freezing procedure to be concurrent.
To summarize, please implement a batch-style freezing interface, which would be helpful for the island working mode.

draft:
```kotlin
fun setAppHidden(hidden: Boolean, vararg packageName: String): Boolean
```
